darasimiolaifa/amistad
# feat(authentication): implement login UI page

## What does this PR do?
This PR implements the login UI view of the application

## What major activities were carried out?
- create login form component
- create form handlers to handle different events from the form
- style form components

## How can this be manually tested?
- Clone the repository
- Fetch the branch
- Install the dependencies
- cd into the `amistad-client` folder
- Start project by running `npm start` in the terminal
- Your browser should automatically load to show you the homepage screen below:

![homepage-with-scream-cards](https://user-images.githubusercontent.com/28527393/71390146-6d5e3480-25ff-11ea-9526-7987d9b60d1f.png)
- In the browser, add `/login` to the url. You should see the following page:

![Login page](https://user-images.githubusercontent.com/28527393/71513730-df52aa00-289b-11ea-80c1-80f469b12308.png)

- In the form, enter `user1@gmail.com` as email, and, `123456` as password. The browser should redirect to take you to the homepage with `screams`.
